@types/express/index.d.ts

```diff
@@ -1,3 +1,6 @@
+import User from '../../src/model/users.model'
+import UserCourse from '../../src/model/user-course.model'
```

Comment:
These imports can be deleted as they're going unused. Normally the compiler would yell about these, but because this file is not compiled it's unaware of their inclusion.

Reply:
Removed